Airports/EGLC/Sector.txt

```diff
@@ -32,6 +32,7 @@ DISPLAY:Only LCAPP:Only LCAPP:LCTWR
 DISPLAY:Only Thames:Only Thames:LCTWR
 DISPLAY:LCTWR:Only LCAPP:LCTWR
 DISPLAY:LCTWR:Only Thames:LCTWR
```

Comment:
We can probably get rid of these two as well but I didn't want to suggest it if it breaks things